### PR TITLE
op-deployer: Add support for inspecting l2 semvers

### PR DESCRIPTION
**Pull request by mslipper**

### CODE REVIEW

File: op-deployer/pkg/deployer/inspect/flags.go [hunk 1]
```diff
@@ -61,6 +61,14 @@ var Commands = []*cli.Command{
 		Action:    DeployConfigCLI,
 		Flags:     Flags,
 	},
+	{
+		Name:      "l2-semvers",
```
Comment (mds1) [inline op-deployer/pkg/deployer/inspect/flags.go:65]: should we rename to `semvers` and output L1 also?

Comment (mslipper) [reply op-deployer/pkg/deployer/inspect/flags.go:65]: I think it's nice having separate commands for L1 and L2 - people may instrument this in other workflows.

File: op-deployer/pkg/deployer/inspect/semvers.go [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+
+	type contractToCheck struct {
+		Address common.Address
+		Name    string
+	}
+
+	contractsOutput := make(map[string]string)
+
+	// The gov token and the proxy admin do not have semvers.
+	contracts := []contractToCheck{
```
Comment (mds1) [inline op-deployer/pkg/deployer/inspect/semvers.go:88]: Looks like we are also not checking the ERC20 and ERC721 tokens?

Comment (mslipper) [reply op-deployer/pkg/deployer/inspect/semvers.go:88]: you mean the mintable ones? those are checked below

Comment (mds1) [reply op-deployer/pkg/deployer/inspect/semvers.go:88]: nevermind yea I see them below, separate from the loop

**Review by mds1 — APPROVED** [review]

**Pull request merged**